index.php

```diff
@@ -842,7 +842,7 @@ function drawmap() {
                     strategies: [bboxStrategyWikipedia],
                     protocol: poiLayerWikipediaHttp
                 });
-                layer_ais = new OpenLayers.Layer.TMS("Marinetraffic", "http://tiles.marinetraffic.com/ais_helpers/shiptilesingle.aspx?output=png&sat=1&grouping=shiptype&tile_size=512&legends=1&zoom=${z}&X=${x}&Y=${y}",
+                layer_ais = new OpenLayers.Layer.TMS("Marinetraffic", "https://tiles.marinetraffic.com/ais_helpers/shiptilesingle.aspx?output=png&sat=1&grouping=shiptype&tile_size=512&legends=1&zoom=${z}&X=${x}&Y=${y}",
```

Comment:
I would recommend changing this to `//tiles.marinetraffic.com...` to prevent mix content problems.

Reply:
There would be mixed content if the map page would be https, but loading a resource (tiles) in plain http, not the other way around.

Protocol relative url ("//...") makes no sense here as even if it starts with http it is redirected to https, just making an extra request before loading the tile.

Reply:
stimmt